__tests__/check-linked-issue-labels.test.ts

```diff
@@ -0,0 +1,94 @@
+import {describe, expect, it} from '@jest/globals'
+import {checkLinkedIssueLabels} from '../src/check-linked-issue-labels'
+
+function createFakeQueryResults(...issuesLabels: string[][]) {
+  return issuesLabels.map((issueLabels, i) => ({
+    node: {
+      labels: {
+        edges: issueLabels.map(label => ({node: {name: label}}))
+      },
+      number: i + 1
+    }
+  }))
+}
+
+describe('checkLinkedIssueLabels', () => {
```

Comment:
it should pass easily, but I'd like to add the "unconfigured" case. This worked for me locally:
```javascript
  it('passes when no required labels are present', () => {
    const check = checkLinkedIssueLabels(createFakeQueryResults(['yolo']), [])
    expect(check).toEqual([])
  })
```